aten/src/ATen/cuda/CublasHandlePool.cpp

```diff
@@ -41,6 +41,13 @@ cublasHandle_t getCurrentCUDABlasHandle() {
   auto handle = myPoolWindow->reserve(device);
   auto stream = c10::cuda::getCurrentCUDAStream();
   TORCH_CUDABLAS_CHECK(cublasSetStream(handle, stream));
+#if CUDA_VERSION >= 11000
+  if (at::globalContext().useTF32CuBLAS()) {
+    TORCH_CUDABLAS_CHECK(cublasSetMathMode(handle, CUBLAS_TF32_TENSOR_OP_MATH));
```

Comment:
What happens if we compiled with cuda 11, but call `cublasSetMathMode(handle, CUBLAS_TF32_TENSOR_OP_MATH)` on a device that doesn't support TF32?  Does it error?  Does it fall back to `CUBLAS_DEFAULT_MATH` behavior?

To avoid ambiguity, maybe the conditional should be
```
if (at::globalContext().useTF32CuBLAS() && at::cuda::getCurrentDeviceProperties()->major >= 8.0) { 
```

Reply:
This should not error out and use a fallback. Same for devices, which support TF32, but where the particular workload might not have been implemented.

Reply:
I believe this is resolved.